server/tests/src/main/scala/sttp/tapir/server/tests/ServerBasicTests.scala

```diff
@@ -261,6 +261,15 @@ class ServerBasicTests[F[_], ROUTE](
           r.body should include("file2:daisy luigi")
         }
     },
+    testServer(in_raw_multipart_out_string, "with error")((_: Seq[Part[Array[Byte]]]) => pureResult(Left(()))) { baseUri =>
+      val file1 = writeToFile("peach mario")
+      basicStringRequest
+        .post(uri"$baseUri/api/echo/multipart")
+        .header(HeaderNames.ContentType, sttp.model.MediaType.ApplicationJson.toString)
+        .multipartBody(multipartFile("file1", file1).fileName("file1.txt"))
+        .send(backend)
+        .map(r => r.code should (be (StatusCode.BadRequest) or be (StatusCode.InternalServerError))) // Some backends interpret the error differently
```

Comment:
we should probably standaradize here and at least create an issue to fix to the desired behavior. As I understand we are sending a json request to something that expects a multipart?